Index: build/tmp/jar/MANIFEST.MF
===================================================================
diff --git a/build/tmp/jar/MANIFEST.MF b/build/tmp/jar/MANIFEST.MF
deleted file mode 100644
--- a/build/tmp/jar/MANIFEST.MF	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
+++ /dev/null	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
@@ -1,2 +0,0 @@
-Manifest-Version: 1.0
-
Index: build/kotlin/ElementFX10SNAPSHOTjar-classes.txt
===================================================================
diff --git a/build/kotlin/ElementFX10SNAPSHOTjar-classes.txt b/build/kotlin/ElementFX10SNAPSHOTjar-classes.txt
deleted file mode 100644
--- a/build/kotlin/ElementFX10SNAPSHOTjar-classes.txt	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
+++ /dev/null	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
@@ -1,1 +0,0 @@
-D:\IDEA_WorkSpace\ElementFX\build\classes\kotlin\main\ink\bluecloud\cloudtools\css\CssResources.class;D:\IDEA_WorkSpace\ElementFX\build\classes\kotlin\main\ink\bluecloud\cloudtools\css\CssResourcesKt.class;D:\IDEA_WorkSpace\ElementFX\build\classes\kotlin\main\ink\bluecloud\cloudtools\css\ElementButton.class;D:\IDEA_WorkSpace\ElementFX\build\classes\kotlin\main\ink\bluecloud\cloudtools\css\ElementButtonKt.class
\ No newline at end of file
Index: .gradle/buildOutputCleanup/cache.properties
===================================================================
diff --git a/.gradle/buildOutputCleanup/cache.properties b/.gradle/buildOutputCleanup/cache.properties
deleted file mode 100644
--- a/.gradle/buildOutputCleanup/cache.properties	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
+++ /dev/null	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
@@ -1,2 +0,0 @@
-#Sat Jul 02 22:54:59 CST 2022
-gradle.version=7.4.2
Index: .idea/jarRepositories.xml
===================================================================
diff --git a/.idea/jarRepositories.xml b/.idea/jarRepositories.xml
deleted file mode 100644
--- a/.idea/jarRepositories.xml	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
+++ /dev/null	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
@@ -1,45 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="RemoteRepositoriesConfiguration">
-    <remote-repository>
-      <option name="id" value="central" />
-      <option name="name" value="Maven Central repository" />
-      <option name="url" value="https://repo1.maven.org/maven2" />
-    </remote-repository>
-    <remote-repository>
-      <option name="id" value="jboss.community" />
-      <option name="name" value="JBoss Community repository" />
-      <option name="url" value="https://repository.jboss.org/nexus/content/repositories/public/" />
-    </remote-repository>
-    <remote-repository>
-      <option name="id" value="MavenRepo" />
-      <option name="name" value="MavenRepo" />
-      <option name="url" value="https://repo.maven.apache.org/maven2/" />
-    </remote-repository>
-    <remote-repository>
-      <option name="id" value="MavenLocal" />
-      <option name="name" value="MavenLocal" />
-      <option name="url" value="file:/$MAVEN_REPOSITORY$/" />
-    </remote-repository>
-    <remote-repository>
-      <option name="id" value="maven2" />
-      <option name="name" value="maven2" />
-      <option name="url" value="https://maven.aliyun.com/repository/central/" />
-    </remote-repository>
-    <remote-repository>
-      <option name="id" value="maven3" />
-      <option name="name" value="maven3" />
-      <option name="url" value="https://maven.aliyun.com/repository/public/" />
-    </remote-repository>
-    <remote-repository>
-      <option name="id" value="maven" />
-      <option name="name" value="maven" />
-      <option name="url" value="https://repo.huaweicloud.com/repository/maven/" />
-    </remote-repository>
-    <remote-repository>
-      <option name="id" value="maven4" />
-      <option name="name" value="maven4" />
-      <option name="url" value="https://oss.sonatype.org/content/repositories/snapshots" />
-    </remote-repository>
-  </component>
-</project>
\ No newline at end of file
Index: .idea/gradle.xml
===================================================================
diff --git a/.idea/gradle.xml b/.idea/gradle.xml
deleted file mode 100644
--- a/.idea/gradle.xml	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
+++ /dev/null	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="GradleMigrationSettings" migrationVersion="1" />
-  <component name="GradleSettings">
-    <option name="linkedExternalProjectsSettings">
-      <GradleProjectSettings>
-        <option name="distributionType" value="WRAPPED" />
-        <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleHome" value="$PROJECT_DIR$/../../Gradle" />
-        <option name="modules">
-          <set>
-            <option value="$PROJECT_DIR$" />
-          </set>
-        </option>
-      </GradleProjectSettings>
-    </option>
-  </component>
-</project>
\ No newline at end of file
Index: .idea/compiler.xml
===================================================================
diff --git a/.idea/compiler.xml b/.idea/compiler.xml
deleted file mode 100644
--- a/.idea/compiler.xml	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
+++ /dev/null	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="CompilerConfiguration">
-    <bytecodeTargetLevel target="17" />
-  </component>
-</project>
\ No newline at end of file
Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
+++ /dev/null	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
@@ -1,8 +0,0 @@
-# 默认忽略的文件
-/shelf/
-/workspace.xml
-# 基于编辑器的 HTTP 客户端请求
-/httpRequests/
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
Index: build/resources/main/cloud_css/ElementButton.css
===================================================================
diff --git a/build/resources/main/cloud_css/ElementButton.css b/build/resources/main/cloud_css/ElementButton.css
deleted file mode 100644
--- a/build/resources/main/cloud_css/ElementButton.css	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
+++ /dev/null	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
@@ -1,111 +0,0 @@
-.white-button {
-    -fx-font-size: 15px;
-    -fx-cursor: HAND;
-    -fx-background-color: rgba(255, 255, 255, 1);
-    -fx-border-color: rgba(220, 223, 230, 1) rgba(220, 223, 230, 1) rgba(220, 223, 230, 1) rgba(220, 223, 230, 1);
-    -fx-border-radius: 5px 5px 5px 5px;
-    -fx-background-radius: 5px 5px 5px 5px;
-}
-.white-button:hover {
-    -fx-background-color: white;
-    -fx-border-color: rgba(198, 226, 255, 1) rgba(198, 226, 255, 1) rgba(198, 226, 255, 1) rgba(198, 226, 255, 1);
-    -fx-text-fill: rgba(64, 158, 255, 1);
-}
-.white-button:pressed {
-    -fx-border-color: rgba(64, 158, 255, 1) rgba(64, 158, 255, 1) rgba(64, 158, 255, 1) rgba(64, 158, 255, 1);
-}
-.default-button {
-    -fx-font-size: 15px;
-    -fx-cursor: HAND;
-    -fx-background-color: rgba(236, 245, 255, 1);
-    -fx-background-radius: 5px 5px 5px 5px;
-    -fx-border-color: rgba(179, 216, 255, 1) rgba(179, 216, 255, 1) rgba(179, 216, 255, 1) rgba(179, 216, 255, 1);
-    -fx-border-radius: 5px 5px 5px 5px;
-    -fx-text-fill: rgba(64, 158, 255, 1);
-}
-.default-button:hover {
-    -fx-background-color: rgba(64, 158, 255, 1);
-    -fx-border-color: rgba(64, 158, 255, 1) rgba(64, 158, 255, 1) rgba(64, 158, 255, 1) rgba(64, 158, 255, 1);
-    -fx-text-fill: rgba(255, 255, 255, 1);
-}
-.default-button:pressed {
-    -fx-background-color: rgba(58, 142, 230, 1);
-    -fx-border-color: rgba(58, 142, 230, 1) rgba(58, 142, 230, 1) rgba(58, 142, 230, 1) rgba(58, 142, 230, 1);
-    -fx-text-fill: rgba(255, 255, 255, 1);
-}
-.green-button {
-    -fx-font-size: 15px;
-    -fx-cursor: HAND;
-    -fx-background-color: rgba(240, 249, 235, 1);
-    -fx-background-radius: 5px 5px 5px 5px;
-    -fx-border-color: rgba(179, 225, 157, 1) rgba(179, 225, 157, 1) rgba(179, 225, 157, 1) rgba(179, 225, 157, 1);
-    -fx-border-radius: 5px 5px 5px 5px;
-    -fx-text-fill: rgba(113, 198, 71, 1);
-}
-.green-button:hover {
-    -fx-background-color: rgba(103, 194, 58, 1);
-    -fx-border-color: rgba(103, 194, 58, 1) rgba(103, 194, 58, 1) rgba(103, 194, 58, 1) rgba(103, 194, 58, 1);
-    -fx-text-fill: rgba(255, 255, 255, 1);
-}
-.green-button:pressed {
-    -fx-background-color: rgba(82, 155, 46, 1);
-    -fx-border-color: rgba(82, 155, 46, 1) rgba(82, 155, 46, 1) rgba(82, 155, 46, 1) rgba(82, 155, 46, 1);
-    -fx-text-fill: rgba(255, 255, 255, 1);
-}
-.gray-button {
-    -fx-font-size: 15px;
-    -fx-cursor: HAND;
-    -fx-background-color: rgba(244, 244, 245, 1);
-    -fx-background-radius: 5px 5px 5px 5px;
-    -fx-border-color: rgba(200, 201, 204, 1) rgba(200, 201, 204, 1) rgba(200, 201, 204, 1) rgba(200, 201, 204, 1);
-    -fx-border-radius: 5px 5px 5px 5px;
-    -fx-text-fill: rgba(153, 156, 161, 1);
-}
-.gray-button:hover {
-    -fx-background-color: rgba(144, 147, 153, 1);
-    -fx-border-color: rgba(144, 147, 153, 1) rgba(144, 147, 153, 1) rgba(144, 147, 153, 1) rgba(144, 147, 153, 1);
-    -fx-text-fill: rgba(255, 255, 255, 1);
-}
-.gray-button:pressed {
-    -fx-background-color: rgba(115, 118, 122, 1);
-    -fx-border-color: rgba(115, 118, 122, 1) rgba(115, 118, 122, 1) rgba(115, 118, 122, 1) rgba(115, 118, 122, 1);
-    -fx-text-fill: rgba(255, 255, 255, 1);
-}
-.orange-button {
-    -fx-font-size: 15px;
-    -fx-cursor: HAND;
-    -fx-background-color: rgba(253, 246, 236, 1);
-    -fx-background-radius: 5px 5px 5px 5px;
-    -fx-border-color: rgba(243, 209, 158, 1) rgba(243, 209, 158, 1) rgba(243, 209, 158, 1) rgba(243, 209, 158, 1);
-    -fx-border-radius: 5px 5px 5px 5px;
-    -fx-text-fill: rgba(234, 175, 86, 1);
-}
-.orange-button:hover {
-    -fx-background-color: rgba(230, 162, 60, 1);
-    -fx-border-color: rgba(230, 162, 60, 1) rgba(230, 162, 60, 1) rgba(230, 162, 60, 1) rgba(230, 162, 60, 1);
-    -fx-text-fill: rgba(255, 255, 255, 1);
-}
-.orange-button:pressed {
-    -fx-background-color: rgba(184, 130, 48, 1);
-    -fx-border-color: rgba(184, 130, 48, 1) rgba(184, 130, 48, 1) rgba(184, 130, 48, 1) rgba(184, 130, 48, 1);
-    -fx-text-fill: rgba(255, 255, 255, 1);
-}
-.red-button {
-    -fx-font-size: 15px;
-    -fx-cursor: HAND;
-    -fx-background-color: rgba(254, 240, 240, 1);
-    -fx-background-radius: 5px 5px 5px 5px;
-    -fx-border-color: rgba(250, 182, 182, 1) rgba(250, 182, 182, 1) rgba(250, 182, 182, 1) rgba(250, 182, 182, 1);
-    -fx-border-radius: 5px 5px 5px 5px;
-    -fx-text-fill: rgba(245, 113, 113, 1);
-}
-.red-button:hover {
-    -fx-background-color: rgba(245, 108, 108, 1);
-    -fx-border-color: rgba(245, 108, 108, 1) rgba(245, 108, 108, 1) rgba(245, 108, 108, 1) rgba(245, 108, 108, 1);
-    -fx-text-fill: rgba(255, 255, 255, 1);
-}
-.red-button:pressed {
-    -fx-background-color: rgba(196, 86, 86, 1);
-    -fx-border-color: rgba(196, 86, 86, 1) rgba(196, 86, 86, 1) rgba(196, 86, 86, 1) rgba(196, 86, 86, 1);
-    -fx-text-fill: rgba(255, 255, 255, 1);
-}
\ No newline at end of file
Index: .idea/misc.xml
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
deleted file mode 100644
--- a/.idea/misc.xml	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
+++ /dev/null	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ExternalStorageConfigurationManager" enabled="true" />
-  <component name="FrameworkDetectionExcludesConfiguration">
-    <file type="web" url="file://$PROJECT_DIR$" />
-  </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_17" default="true" project-jdk-name="ibm-17" project-jdk-type="JavaSDK">
-    <output url="file://$PROJECT_DIR$/out" />
-  </component>
-</project>
\ No newline at end of file
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
deleted file mode 100644
--- a/.idea/vcs.xml	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
+++ /dev/null	(revision 67ba65b5c8660d0938cde148fc052c477cb0bc17)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$" vcs="Git" />
-  </component>
-</project>
\ No newline at end of file
diff --git a/.gradle/7.4.2/dependencies-accessors/gc.properties b/.gradle/7.4.2/dependencies-accessors/gc.properties
deleted file mode 100644
diff --git a/.gradle/vcs-1/gc.properties b/.gradle/vcs-1/gc.properties
deleted file mode 100644
diff --git a/.gradle/7.4.2/gc.properties b/.gradle/7.4.2/gc.properties
deleted file mode 100644
